sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -876,7 +876,6 @@ class MapboxMap :
    * @return Returns the distance measured in meters.
    */
   override fun getMetersPerPixelAtLatitude(latitude: Double, zoom: Double): Double {
-    checkNativeMap("getMetersPerPixelAtLatitude")
```

Comment:
why these are removed ?

Reply:
Those are static gl-native methods used for calculation, no need to request them on main thread only. Additionally `unproject` method is used in custom evaluator `flyTo` implementation and will be always called on background thread (if animator started from background thread) without ability to re-post on main as it expects the result in the same callchain.